### PR TITLE
Initial integration tests

### PR DESCRIPTION
**Pull request by BlakeDonn**
### What does this PR do?

* Adds support for integration testing
* Adds integrations testing up to the final loading animation

### Where should the reviewer start?

* Start by checking if the App tests are robust enough 
 
### How should this be manually tested?

* Pull branch, npm install, ensure tests are passing
 
### Any background context you want to provide?

* Needed a slight adjustment for the Spring component to function
* Current need to find a workaround for the final animation, which breaking integration tests

### What are the relevant tickets?

progresses #56

### CODE REVIEW

**Review by timkeresey — APPROVED** [review]

File: .vscode/launch.json [hunk 1]
```diff
@@ -0,0 +1,15 @@
+{
+  // Use IntelliSense to learn about possible attributes.
+  // Hover to view descriptions of existing attributes.
+  // For more information, visit: https://go.microsoft.com/fwlink/?linkid=830387
+  "version": "0.2.0",
```
Comment (Atos20) [inline .vscode/launch.json:5]: I don't think we need this file, we can remove it.

File: package.json [hunk 2]
```diff
@@ -16,6 +16,9 @@
     "react-icons": "^4.1.0",
     "react-router-dom": "^5.2.0",
     "react-scripts": "4.0.1",
+    "react-spring": "^8.0.27",
```
Comment (Atos20) [inline package.json:19]: These are the animation dependencies

File: src/components/App/App.test.tsx [hunk 3]
```diff
@@ -1,13 +1,74 @@
...
+import { MemoryRouter } from "react-router-dom";
+import "@testing-library/jest-dom";
 import App from './App';
 
-test('renders learn react link', () => {
-  render(<App />);
-  const linkElement = screen.getByText(/learn react/i);
-  expect(linkElement).toBeInTheDocument();
-});
+describe("App", () => {
```
Comment (Atos20) [inline src/components/App/App.test.tsx:8]: nice integration!

File: src/components/Home/Home.css [hunk 4]
```diff
@@ -32,11 +32,28 @@
   z-index: 1;
   /* margin: 3%; */
 }
+.banner-bx{
```
Comment (Atos20) [inline src/components/Home/Home.css:35]: This styles the banner we were missing in the home page

**Review by Atos20 — APPROVED** [review]

**Pull request merged**